platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -47,67 +47,42 @@
   public static final String HTTP_PROTOCOL = "http://";
 
   private static final String DEFAULT_LOCALHOST_DN = "localhost.local";
-
-  private MBeanServer mbeanServer;
-
-  private ObjectName objectName;
-
-  private String oldHostName = SystemBaseUrl.getHost();
-
   private static final String KARAF_ETC = "karaf.etc";
-
   private static final String LOCAL_HOST = "localhost";
-
   private static final String SYSTEM_PROPERTIES_FILE = "system.properties";
-
   private static final String USERS_PROPERTIES_FILE = "users.properties";
-
   private static final String USERS_ATTRIBUTES_FILE = "users.attributes";
-
   private static final String HOST_TITLE = "Host";
-
   private static final String HOST_DESCRIPTION =
       "The host name or IP address used to advertise the system. Possibilities include the address of a single node of that of a load balancer in a multi-node deployment. NOTE: This setting will take effect after a system restart.";
-
   private static final ArrayList<String> PROTOCOL_OPTIONS = new ArrayList<>();
-
-  static {
-    PROTOCOL_OPTIONS.add("https://");
-    PROTOCOL_OPTIONS.add("http://");
-  }
-
   private static final String HTTP_PORT_TITLE = "HTTP Port";
-
   private static final String HTTP_PORT_DESCRIPTION =
       "The port used to advertise the system. Possibilities include the port of a single node of that of a load balancer in a multi-node deployment. NOTE: This setting will take effect after a system restart.";
-
   private static final String HTTPS_PORT_TITLE = "HTTPS Port";
-
   private static final String HTTPS_PORT_DESCRIPTION =
       "The secure port used to advertise the system. Possibilities include the port of a single node of that of a load balancer in a multi-node deployment. NOTE: This setting will take effect after a system restart.";
-
   private static final String ORGANIZATION_TITLE = "Organization";
-
   private static final String ORGANIZATION_DESCRIPTION =
       "The name of the organization that runs this instance.";
-
   private static final String SITE_CONTACT_TITLE = "Site Contact";
-
   private static final String SITE_CONTACT_DESCRIPTION = "The email address of the site contact.";
-
   private static final String SITE_NAME_TITLE = "Site Name";
-
   private static final String SITE_NAME_DESCRIPTION =
       "The unique name of this instance. This name will be provided via web services that ask for the name.";
-
   private static final String VERSION_TITLE = "Version";
-
   private static final String VERSION_DESCRIPTION = "The version of this instance.";
-
   private static final Logger LOGGER = LoggerFactory.getLogger(SystemPropertiesAdmin.class);
-
   private static final ObjectMapper MAPPER = JsonFactory.create();
 
+  static {
+    PROTOCOL_OPTIONS.add("https://");
+    PROTOCOL_OPTIONS.add("http://");
```

Comment:
:question: Should we use HTTP_PROTOCOL (https://github.com/codice/ddf/pull/2768/files#diff-15df2753c51e8eecc8b9aa415a5f3092R47) here?  Do we need something similar for https?